src/urllib3/util/timeout.py

```diff
@@ -196,7 +206,7 @@ def start_connect(self):
         self._start_connect = current_time()
```

Comment:
We can actually use `time.monotonic` instead of `current_time` here (and elsewhere) because it's [always available in Python 3.5+](https://docs.python.org/3/library/time.html#time.monotonic)

Reply:
Here is the [PR](https://github.com/urllib3/urllib3/pull/2214)